QUARKUS-6258.md

```diff
@@ -47,6 +54,10 @@ Primary focus will be on RHEL-based testing to ensure functionality of proposed
 
 As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.d testing.
 
+#### Impact on test suites for Full Support
```

Comment:
On line 53 you write `Primary focus will be on RHEL-based testing to ensure functionality of proposed features against productized bits. For the OpenShift side, we will explore the possibility of using the @GitRepositoryQuarkusApplication approach.`, what does it mean "primary focus", does it mean you don't test anything else or how much? Such a soft metrics are acceptable, but it would be nice to mention how non-primary focus looks like?

Reply:
why is "Impact on test suite" subsection of the "Existing test coverage"?

Reply:
Okay, that is not how I understood "non-primary", but unless someone else have same issue, I am fine with keeping it as is. Thanks